retropie_packages.sh

```diff
@@ -73,5 +73,5 @@ fi
 printMsgs "console" "${__INFMSGS[@]}"
 
 if [[ -n $__joy2key_pid ]]; then
-    kill -INT $__joy2key_pid 2>/dev/null
+    kill -INT $__joy2key_pid 2>/dev/null || echo "Failed to kill joy2key"
```

Comment:
I could be mistaken, but without the || the entire script's exit code will be non-zero if the kill is non-zero, which causes my automation to fail.